concent_api/core/subtask_helpers.py

```diff
@@ -185,3 +187,28 @@ def are_subtask_results_accepted_messages_signed_by_the_same_requestor(subtask_r
         ) for subtask_results_accepted in subtask_results_accepted_list
     )
     return are_all_signed_by_requestor
+
+
+def get_one_or_none_subtask_from_database(
+    subtask_id: str,
+    optional_condition_dict: Dict[str, str] = None,
+    lock_returned_subtasks_in_database: bool = False,
```

Comment:
to resolve this problem this is not needed. I did it to make this function more general- to get subtask without obligatory locking, with this function. And False as default was, because I think locking database should be written explicitly, not by hidden default parametr. Removed